service/debugger/debugger.go

```diff
@@ -1310,6 +1310,16 @@ func (d *Debugger) ListDynamicLibraries() []api.Image {
 	return r
 }
 
+// ExamineMemory returns a list of specific address memory that len is num.
+func (d *Debugger) ExamineMemory(address uintptr, num int) ([]byte, error) {
+	d.processMutex.Lock()
+	defer d.processMutex.Unlock()
+
+	thread := d.target.CurrentThread()
+
+	return proc.ExamineMemory(thread, address, num)
```

Comment:
`proc.ExamineMemory` is only used here, it should be inlined here instead of being a separate function.

Reply:
Done